connect-location/src/main/java/com/ifttt/location/LocationInfo.java

```diff
@@ -0,0 +1,47 @@
+package com.ifttt.location;
+
+import com.squareup.moshi.Json;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+import java.util.UUID;
+
+final class LocationInfo {
+
+    private static final DateFormat LOCATION_EVENT_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'",
+        Locale.US
```

Comment:
is this the correct format? Should it be `yyyy-MM-dd HH:mm:ss zz`?

Reply:
They are both valid format, this one seems to be more inline with the standard ISO 8601.